<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Controller</class>
 <widget class="QMainWindow" name="Controller">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1105</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1105</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1105</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Torchlight - Snake AI</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>logo.png</normaloff>logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>802</width>
      <height>481</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="boardLayout">
     <item>
      <widget class="QGraphicsView" name="graphicsView">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>795</width>
         <height>465</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>795</width>
         <height>465</height>
        </size>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="dragMode">
        <enum>QGraphicsView::ScrollHandDrag</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>280</y>
      <width>261</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="btnsLayout">
     <item>
      <widget class="QPushButton" name="btnInit">
       <property name="font">
        <font>
         <family>Ubuntu Mono</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Play</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAuto">
       <property name="font">
        <font>
         <family>Ubuntu Mono</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Auto Play</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnReset">
       <property name="font">
        <font>
         <family>Ubuntu Mono</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnTrain">
       <property name="font">
        <font>
         <family>Ubuntu Mono</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Train</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPlayIA">
       <property name="font">
        <font>
         <family>Ubuntu Mono</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Play IA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnExit">
       <property name="font">
        <font>
         <family>Ubuntu Mono</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>50</y>
      <width>261</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="neuralLayout">
     <item>
      <layout class="QHBoxLayout" name="datasetLayout">
       <item>
        <widget class="QLabel" name="lblInputs">
         <property name="font">
          <font>
           <family>Ubuntu Mono</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>DATASET SIZE:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblInputsSize">
         <property name="font">
          <font>
           <family>Ubuntu Mono</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="meanErrorLayout">
       <item>
        <widget class="QLabel" name="lblMeanErr">
         <property name="font">
          <font>
           <family>Ubuntu Mono</family>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>MEAN ERROR:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblMeanError">
         <property name="font">
          <font>
           <family>Ubuntu Mono</family>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="actEpochLayout">
       <item>
        <widget class="QLabel" name="lblEpocs">
         <property name="font">
          <font>
           <family>Ubuntu Mono</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>EPOCH:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblActEpoch">
         <property name="font">
          <font>
           <family>Ubuntu Mono</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="directionLayout">
       <item>
        <widget class="QLabel" name="lblDirection">
         <property name="font">
          <font>
           <family>Ubuntu Mono</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>DIRECTION:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblActDirection">
         <property name="font">
          <font>
           <family>Ubuntu Mono</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="inputEpochsLayout">
       <item>
        <widget class="QLabel" name="lblInputEpoch">
         <property name="font">
          <font>
           <family>Ubuntu Mono</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>EPOCHS:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="inputEpoch">
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>30</height>
          </size>
         </property>
         <property name="locale">
          <locale language="Spanish" country="Spain"/>
         </property>
         <property name="placeholderText">
          <string>10 000</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>801</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="dataLayout">
     <item>
      <layout class="QHBoxLayout" name="scoreLayout">
       <item>
        <widget class="QLabel" name="scoreLabel">
         <property name="font">
          <font>
           <family>Ubuntu Mono</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>SCORE:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="scoreLabelNum">
         <property name="font">
          <font>
           <family>Ubuntu Mono</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="lblSnakeX">
       <property name="font">
        <font>
         <family>Ubuntu Mono</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>SNAKE X:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblSnakeY">
       <property name="font">
        <font>
         <family>Ubuntu Mono</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>SNAKE Y:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblFruitX">
       <property name="font">
        <font>
         <family>Ubuntu Mono</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>FRUIT X:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblFruitY">
       <property name="font">
        <font>
         <family>Ubuntu Mono</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>FRUIT Y:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
